docs/UserGuide.md

```diff
@@ -680,7 +686,7 @@ For example:
 Other examples:
 * `find-class CS2030 lab`<br/>
   `delete-class 1`<br/>
-  Deletes the 1st class in the result of the `find-class` command.
+  Deletes the `CS2030 lab` class.
```

Comment:
Is this in the context of the sample data given? Otherwise it could be misleading.

Same concern for the rest of the edits for `Other examples`.